CodingPractice/PythonAssignments/shoppingcart/dao/postgres/ProductPostgresDao.py

```diff
@@ -61,3 +61,23 @@ def create_product(self, product_dto):
                       v)
         with closing(self._postgres_conn.cursor()) as cursor:
             cursor.execute(self.INSERT_SQL, prod_tuple)
+
+    def delete_product(self, productDto):
+        product_id = productDto.get_id()
+        with closing(self._postgres_conn.cursor()) as cursor:
+            cursor.execute(f"DELETE FROM product WHERE product_id={product_id};")
+
+    def update_product(self, productDto):
```

Comment:
I'd suggest that you write a method that deals with turning is_vatable into 0 or 1